src/test/scala/uk/gov/nationalarchives/draftmetadatavalidator/LambdaSpec.scala

```diff
@@ -49,8 +49,12 @@ class LambdaSpec extends ExternalServicesSpec {
 
   "handleRequest" should "download the draft metadata csv file, validate, save empty error file to s3 and save metadata to db if it has no errors" in {
     authOkJson()
-    val fileIdMetadata = testFileIdMetadata(Seq("test/test1.txt", "test/test2.txt", "test/test3.txt"))
-    graphqlOkJson(true, fileIdMetadata)
+    val fileTestData = List(
```

Comment:
This is the same data for all the test. Could this be made a class val instead of hard coded into each of the tests?

Reply:
Yeah, you are right. Created a common test data. Thanks